packages/library-base/src/commonMain/kotlin/io/realm/notifications/ObjectChange.kt

```diff
@@ -47,12 +64,12 @@ import io.realm.RealmObject
  *
  * For state of update changes, a list with the updated field names from the previous version is provided.
  */
-sealed interface ObjectChange<O : RealmObject> {
+sealed interface ObjectChange<O : RealmObject> : QueryObjectChange<O> {
```

Comment:
This seems problematic? Doesn't this force people using `ObjectChange` to also reason about `PendingObject` now?

Reply:
I missed some nuances around how this works. This seems fine 👍